### PR TITLE
Feature/feature pet info

### PR DESCRIPTION
**Pull request by santidrj**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/InfoPetFragment.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213151&bid=17093944)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import com.google.android.material.navigation.NavigationView;
 
 import org.pesmypetcare.mypetcare.R;
+import org.pesmypetcare.mypetcare.activities.fragments.InfoPetFragment;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java]: Codacy Issue found: [Unused import - org.pesmypetcare.mypetcare.activities.fragments.InfoPetFragment.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213151)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/InfoPetFragment.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     * Configure the Dropdown menu of gender
+     */
+    private void setGenderDropdownMenu() {
+        AutoCompleteTextView gender = binding.inputGender;
+        ArrayAdapter<String> adapter = new ArrayAdapter<>(Objects.requireNonNull(getContext()),
+                R.layout.drop_down_menu_item, new String[] {getString(R.string.male), getString(R.string.female)});
+        gender.setAdapter(adapter);
+    }
+
+    /**
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/InfoPetFragment.java]: Codacy Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213151)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java (hunk 1 above)
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java]: Codacy Issue found: [Avoid unused imports such as 'org.pesmypetcare.mypetcare.activities.fragments.InfoPetFragment'](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213151)

**Review by AlbertPG — COMMENTED** [review]
You need to put dots, but Codacy will tell you to fix it. Aside from that, I don't see any problems

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/InfoPetFragment.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213151&bid=17093944)

**Review by AlbertPG — APPROVED** [review]
You have my approval

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by enrichp — APPROVED** [review]

**Pull request merged**